x-pack/plugins/rollup/public/crud_app/services/track_user_action.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import chrome from 'ui/chrome';
+import { getHttp } from './http_provider';
+
+export function trackUserAction(actionType) {
+  getHttp().post(chrome.addBasePath(`/api/user_action/rollup-job-wizard/${actionType}`));
+}
+
+export function trackUserRequest(action, actionType) {
+  // Only track successful actions.
```

Comment:
There will be no use cases where we'll be interested in success/fail ratio on a user request? To me there are 3 telemetries on a request:

- action-requested
- action-succeed
- action-failed

@yaronp68 are we only interested in successes?